infrastructure-management/README.md

```diff
@@ -0,0 +1,11 @@
+# Infrastructure Management
+
+In this project, we will build the infrastructure to run a service in AWS. The infrastructure will be defined in code via Terraform. We will learn how to build, change and destroy our infra in a safe, reproducible way.
+
+Timebox: X days
+
+Learning objectives:
+
+- Create an autoscaling group, security group, load balancer in Terraform
+- Understand how Terraform handles state and fixes discrepancies
```

Comment:
```suggestion
- Handle state with Terraform state files on a remote backend
- Fix state discrepancies with Terraform state locking
```
Instead of saying "understand", let's concretise this by listing steps the learner will take to understand.  (The state locking is just a guess. I don't know what this project entails.)

https://teachtogether.tech/en/#s:process-objectives